From 0140825e5c16b79e3089fa550c550b5a0c664a51 Mon Sep 17 00:00:00 2001
From: kitakar5525 <34676735+kitakar5525@users.noreply.github.com>
Date: Mon, 22 Jul 2019 19:38:37 +0900
Subject: [PATCH 16/28] ipts: uncomment downstream_hpd_needs_d0

Because I'm not sure why this is commented out, I revert this change.
---
 drivers/gpu/drm/i915/display/intel_dp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/display/intel_dp.c b/drivers/gpu/drm/i915/display/intel_dp.c
index f85f6e533..4336df46f 100644
--- a/drivers/gpu/drm/i915/display/intel_dp.c
+++ b/drivers/gpu/drm/i915/display/intel_dp.c
@@ -2970,8 +2970,8 @@ void intel_dp_sink_dpms(struct intel_dp *intel_dp, int mode)
 		return;
 
 	if (mode != DRM_MODE_DPMS_ON) {
-		//if (downstream_hpd_needs_d0(intel_dp))
-		//	return;
+		if (downstream_hpd_needs_d0(intel_dp))
+			return;
 
 		ret = drm_dp_dpcd_writeb(&intel_dp->aux, DP_SET_POWER,
 					 DP_SET_POWER_D3);
-- 
2.22.0

